pkg/lib/bundle/generate.go

```diff
@@ -76,6 +76,24 @@ func GenerateFunc(directory, outputDir, packageName, channels, channelDefault st
 		return err
 	}
 
+	if channels == "" || packageName == "" || channelDefault == "" {
+		I, err := NewBundleDirInterperter(filepath.Join(directory, ".."))
+		if err != nil {
+			return fmt.Errorf("error interpretating bundle from directory %s, %v", directory, err)
```

Comment:
spelling nit: `interpreting`

Also, it feels like the error messaging here is going to be unclear to users. Can we be more descriptive? Right now it seems like we don't differentiate between "this bundle isn't in a directory with our special helpful case" and "you didn't give us enough information to handle this request". For example if they just didn't specify a list of channels in a bundle that's in its own directory, we shouldn't surface "unable to interpret bundle from directory". we should tell them "missing input parameters".

There's also the case where a user just doesn't give us a default channel for example that we were already handling and I think this removes that ability as well